cmd/ontap-cvo.go

```diff
@@ -409,6 +409,18 @@ func processConfigmap(client *kubernetes.Clientset, namespace string, email stri
 		// Create the s3 buckets
 		s3BucketsList := sharesData[k]
 		for _, s := range s3BucketsList {
+			// First verify that the share is not a duplicate
+			err = validateUserInput(client, namespace, svmInfo.Vserver, s)
+			if err != nil {
+				klog.Errorf("Error in requesting shares for: %s", namespace)
+				return &ShareError{
+					ErrorMessage: err.Error(),
+					Svm:          "",
+					Share:        "",
```

Comment:
Svm and Share are two inputs that you can put here. svminfo.Vserver and s(you use them in the function call)